azure/services/managedclusters/spec.go

```diff
@@ -487,7 +487,14 @@ func (s *ManagedClusterSpec) Parameters(ctx context.Context, existing *asocontai
 				Key:  secret.KubeconfigDataName,
 			},
 		},
+		ConfigMaps: &asocontainerservicev1.ManagedClusterOperatorConfigMaps{
+			OIDCIssuerProfile: &genruntime.ConfigMapDestination{
+				Name: oidcIssuerURLConfigMapName(s.ClusterName),
+				Key:  "oidc-issuer-profile-url",
```

Comment:
`"oidc-issuer-profile-url"` should be a const

Reply:
added via fe9c588ee0b05ef1f32b897f23b532fe76994260